worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -0,0 +1,35 @@
+# Dark Souls 3 Randomizer Setup Guide
+
+## Required Software
+
+- [Dark Souls 3](https://store.steampowered.com/app/374320/DARK_SOULS_III/)
+- [Dark Souls 3 AP Client](https://github.com/Marechal-L/DarkSouls3-Archipelago-client)
+
+## General Concept
+
+The Dark Souls 3 AP Client is a dinput8.dll triggered when launching Dark Souls 3. This .dll file will launch a command 
+prompt where you can read information about your run and write any command to interact with the Archipelago server.  
+
+The randomization is performed by the AP.json file, an output file generated by the Archipelago server.
```

Comment:
While this is probably fine in general, two possible ideas to make it easier to get started as a player:
* use slot_data that you get on connect instead of an external json file. - slot data is transferred as json, but you will only get it once connected and slot name is entered, so this may or may not work with the way you hook into the game - or
* use a different file extension (.apds3) and supply a program that will automatically put it in the correct spot with the correct name

Reply:
Using the slot_data is a great idea! But as you said I will need to try if I would be able to hook the game at runtime and not at startup.
I'm starting to be running out of time for this project so I would probably only be able to provide this as an update :-/

Reply:
That's ok, it does not have to be perfect in the first iteration. Just wanted to let you know that those options exist.